src/Conversion/ONNXToKrnl/NN/Normalization.cpp

```diff
@@ -219,15 +214,14 @@ struct ONNXInstanceNormalizationOpLowering : public ConversionPattern {
           ValueRange spatial2_loopDef = create.krnl.defineLoops(rank - 2);
           create.krnl.iterateIE(spatial2_loopDef, spatial2_loopDef, lbs, ubs,
               [&](KrnlBuilder &createKrnl, ValueRange spatial_loopInd) {
-                MathBuilder createMath(createKrnl);
                 SmallVector<Value, 6> inputAccessFct = {
                     n.getValue(), c.getValue()};
                 for (int d = 0; d < rank - 2; ++d)
                   inputAccessFct.emplace_back(spatial_loopInd[d]);
                 // tmp += input[n,c, spatial dims]
-                Value oldSum = createKrnl.load(tmpMemRef, {});
-                Value val = createKrnl.load(inputMemRef, inputAccessFct);
-                Value newSum = createMath.add(oldSum, val);
+                Value oldSum = create.krnl.load(tmpMemRef, {});
```

Comment:
I know that it might work as is, but the model for using builders is to recreate them in each context. Here the create.krml.iterateIE has a lambda functions that provides a KrnlBuilder, and inside that lambda function, that builder should be use to create other builders... So referring here to the external `create.krnl` actually bypass that inner `createKrnl` constructor. 

Please change everywhere accordingly.

Reply:
Great! thanks